Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM quay.io/ukhomeofficedigital/alpine:v3.14 as builder
+
+USER root
+
+RUN apk add --no-cache openjdk11-jre
+
+COPY build/libs/*.jar .
+
+RUN java -Djarmode=layertools -jar *.jar extract
+
+FROM quay.io/ukhomeofficedigital/alpine:v3.14
+
+USER root
```

Comment:
Do you need to specify the `USER root` directive every time?

Reply:
Yeah the documentation isn't super clear on this but from testing it doesn't carry over. From [here](https://docs.docker.com/develop/develop-images/multistage-build/) "The second FROM instruction starts a new build stage with the alpine:latest image as its base. The COPY --from=0 line copies just the built artifact from the previous stage into this new stage. The Go SDK and any intermediate artifacts are left behind, and not saved in the final image." suggests everything is left behind unless explicitly specified